pipes/rules/metagenomics.rules

```diff
@@ -29,16 +30,29 @@ rule kraken:
     resources: cores=int(config.get("number_of_threads", 1))
     params: numThreads=str(config.get("number_of_threads", 1))
     run:
+        shell("pyenv which python")
+        shell("echo \n\n\n\n\n\n\n")
         shell("{config[bin_dir]}/metagenomics.py kraken {input} {config[kraken_db]} --outReads {output.reads} --outReport {output.report} --numThreads {params.numThreads}")
 
-rule krona_import_taxonomy:
-    input: join(config["data_dir"], config["subdirs"]["metagenomics"], "{sample}.kraken.reads.gz")
-    output: join(config["data_dir"], config["subdirs"]["metagenomics"], "{sample}.kraken.krona.html")
-    shell:
-        """
-        {config[bin_dir]}/metagenomics.py krona {input} {config[krona_db]} {output} --noRank
-        """
+method_reads_ext = {
+    'diamond': 'diamond.lca.gz',
+    'kraken': 'kraken.reads.gz'
+}
 
+method_krona_columns = {
+    'diamond':  (1, 2),
```

Comment:
I changed this slightly but keeping it because the output formats are not really compatible - the kraken output has extra columns whose meaning cannot be translated for diamond.